influxql/ast.go

```diff
@@ -3034,6 +3040,14 @@ type NumberLiteral struct {
 // String returns a string representation of the literal.
 func (l *NumberLiteral) String() string { return strconv.FormatFloat(l.Val, 'f', 3, 64) }
```

Comment:
If we're making an `IntegerLiteral` then we should rename `NumberLiteral` to `FloatLiteral` otherwise it's going to get confusing.